content/en/continuous_delivery/search/_index.md

```diff
@@ -15,44 +15,42 @@ further_reading:
 CD Visibility is in private beta. To request access, complete the form.
 {{< /callout >}}
 
-## Overview
+## Deployments
 
-The [**Deployments** page][1] is useful for developers who want to keep an eye on the deployments for their environments.
+To see an overview of your deployments, navigate to [**Software Delivery** > **CD Visibility** > **Deployments**][1].
 
-{{< img src="/continuous_delivery/search/deployments.png" text="The Deployments page in Datadog" style="width:100%" >}}
+The [**Deployments** page][1] shows stats aggregated by services and environments over the selected time frame, as well as the status of the latest deployment execution. Use this page to see all your service deployments and get a view of their health.
+The metrics shown include the number of executions and failures, the failure rate, the median duration, and the 95th percentile duration. This information reveals which deployments have a higher probability of failure and which deployments are taking the most time to be executed. The effect of the latest changes can be seen by checking the status, revision and time of the last deployment result.
 
-Access high-level accumulation and trends, including:
+<div class="alert alert-info">Deployments without services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page. You can search for these deployments in the Deployment Executions page: <code>@deployment.partial_deployment:* OR -@deployment.service:*</code>.</div>
```

Comment:
In the first sentence, is it deployments with partial deployment executions that are excluded or deployments without partial deployment executions that are excluded?

Reply:
There are two types of deployment excluded from the page:
1. Partial deployments (i.e. only a subset of steps were executed for a deployment)
2. Deployments without service configured (they might or might not have a service, and in the latter case they are excluded from the aggregated stats)

Reply:
@andrea-mosk Thanks for the explanation! Does this work?

> Deployments with no services configured and those with partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Reply:
Yeah looks good, I would just remove "those that":

> Deployments with no services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Reply:
Changed 😃